app/models/discussion.rb

```diff
@@ -8,10 +8,6 @@ class Discussion < ActiveRecord::Base
 
   default_scope -> { order('updated_at DESC') }
 
-  after_save :update_subscriptions
```

Comment:
From what I could see, this meant two calls to the db. One that saves it, and an update. I don't see the need for that.

> On 13 Jan 2015, at 10:10, Miguel Palhas <notifications@github.com> wrote:
> 
> In app/models/discussion.rb:
> 
> > @@ -8,10 +8,6 @@ class Discussion < ActiveRecord::Base
> >  
> >    default_scope -> { order('updated_at DESC') }
> >  
> > -  after_save :update_subscriptions
> Actually there's no problem here. Even if the callback gets triggered twice in a transaction, the subscription won't change states again, and no new notification will be generated
> 
> —
> Reply to this email directly or view it on GitHub.
>